Packs/ExtFilter/ReleaseNotes/1_1_11.md

```diff
@@ -0,0 +1,5 @@
+
+#### Scripts
+- **ExtFilter**
+  - Enhance an feature of `if-then-else` to make expressions
```

Comment:
Maybe write about the 'lhs' and the functionality that was added

Reply:
- Added the **lhs** parameter to the  **if-then-else** in order to make expressions  (explain here)